Doc/library/asyncio-task.rst

```diff
@@ -103,6 +103,29 @@ To actually run a coroutine, asyncio provides three main mechanisms:
       world
       finished at 17:14:34
 
+* The :class:`asyncio.TaskGroup` class provides a more modern
+  alternative to :func:`create_task`.
+  Using this API the last example becomes::
+
+      async def main():
+          async with asyncio.TaskGroup() as tg:
+              task1 = tg.create_task(
+                  say_after(1, 'hello'))
+
+              task2 = tg.create_task(
+                  say_after(2, 'world'))
+
+              print(f"started at {time.strftime('%X')}")
+
+          # The wait is implicit when the context manager exits.
+
+          print(f"finished at {time.strftime('%X')}")
+
+The timing and output should be the same as for the previous version.
+
+   .. versionchanged:: 3.11
+      Added :class:`asyncio.TaskGroup`.
```

Comment:
Why not

```suggestion
  .. versionadded:: 3.11
     :class:`asyncio.TaskGroup`.
```

Otherwise, just fix indent level

```suggestion
  .. versionchanged:: 3.11
     Added :class:`asyncio.TaskGroup`.
```